modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/definition/tree/manager/test/ObjectDefinitionTreeUtilTest.java

```diff
@@ -1646,6 +1713,132 @@ public void testUpdateNodeObjectDefinitionWithPublishedNodeDescendants()
 			_objectDefinitionLocalService,
 			new String[] {"C_A", "C_AA", "C_AAA", "C_AAAA", "C_AAAAA"},
 			_objectEntryLocalService, _objectRelationshipLocalService);
+
+		// Update a node with published/draft descendant nodes
```

Comment:
I think Brian prefers all written instead of symbols (/).
What do you think about: Update a node with published and draft descendant nodes
and hope that he doesn't want to reorder :grimacing: